# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: pleroma-bot\n"
"POT-Creation-Date: 2021-02-21 20:32+0100\n"
"PO-Revision-Date: 2021-02-21 20:32+0100\n"
"Last-Translator: robertoszek <robertoszek@robertoszek.xyz>\n"
"Language-Team: \n"
"Language: es\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"
"X-Generator: Poedit 2.0.6\n"
"X-Poedit-Basepath: ../../..\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Poedit-SearchPath-0: .\n"
"X-Poedit-SearchPathExcluded-0: tests\n"

#: _pin.py:25
msgid "Pinning post:\t{}"
msgstr "Fijando post:\t{}"

#: _pin.py:56
msgid "Unpinning previous:\t{}"
msgstr "Desfijando anterior:\t{}"

#: _pin.py:60
msgid ""
"File with previous pinned post ID not found or empty. Checking last posts "
"for pinned post..."
msgstr ""
"No se ha encontrado el archivo con el ID del anterior post fijado. Revisando "
"últimos posts..."

#: _pin.py:65
msgid "Pinned post not found. Giving up unpinning..."
msgstr "Post fijado no encontrado. Dejamos de intentar el desfijado..."

#: _pleroma.py:43
msgid "No posts were found in the target Fediverse account"
msgstr "No se han encontrado posts en la cuenta de Fediverso objectivo"

#: _pleroma.py:104
#, python-brace-format
msgid ""
"Exception occurred\n"
"Media size too large:\n"
"Filename: {file}\n"
"Size: {size}MB\n"
"Consider increasing the attachment\n"
" size limit of your instance"
msgstr ""
"Se ha producido una excepción\n"
"Tamaño demasiado grande:\n"
"Nombre de fichero: {file}\n"
"Tamaño: {size}MB\n"
"Considere aumentar el límite de\n"
" tamaño de adjuntos para su instancia"

#: _pleroma.py:119
msgid "Error uploading media:\t{}"
msgstr "Error mientras subiendo contenido:\t{}"

#: _pleroma.py:154
msgid "Post in Pleroma:\t{}"
msgstr "Publicando en Pleroma\t{}"

#: _pleroma.py:210
msgid ""
"Total number of metadata fields cannot exceed 4.\n"
"Provided: {}. Exiting..."
msgstr ""
"El número de campos de metadatos no puede ser superior a 4.\n"
"Se han introducido {}. Saliendo..."

#: _pleroma.py:244
msgid "Updating profile:\t {}"
msgstr "Actualizando perfil:\t{}"

#: _processing.py:22
msgid "Processing tweets... "
msgstr "Procesando tweets... "

#: _processing.py:142
#, python-brace-format
msgid ""
"Exception occurred\n"
"Media not found (404)\n"
"{tweet} - {media_url}\n"
"Ignoring attachment and continuing..."
msgstr ""
"Se ha producido una excepción\n"
"Contenido no encontrado (404)\n"
"{tweet} - {media_url}\n"
"Ignorando el adjunto y continuando..."

#: _processing.py:167
msgid "Attachment exceeded config file size limit ({})"
msgstr "Contenido adjunto ha excedido el límite configurado ({})"

#: _processing.py:172
msgid "File size: {}MB"
msgstr "Tamaño de fichero: {}MB"

#: _processing.py:175
msgid "Ignoring attachment and continuing..."
msgstr "Ignorando adjunto y continuando..."

#: _twitter.py:88
msgid "API version not supported: {}"
msgstr "Versión de API no soportada: {}"

#: _twitter.py:103
msgid "max_tweets must be between 10 and 100. max_tweets: {}"
msgstr "El valor de max_tweets debe estar entre 10 y 100. max_tweets: {}"

#: _twitter.py:206
msgid "Gathering tweets... "
msgstr "Recopilando tweets... "

#: _utils.py:110
msgid "Current pinned:\t{}"
msgstr "Fijado actualmente:\t{}"

#: _utils.py:120
msgid "Previous pinned:\t{}"
msgstr "Fijado anteriormente:\t{}"

#: _utils.py:232
msgid "Instance response was not understood {}"
msgstr "La respuesta de la instancia no ha podido ser interpretada {}"

#: _utils.py:236
msgid "Assuming target instance is Mastodon..."
msgstr "Suponiendo la instancia objetivo es Mastodon..."

#: _utils.py:240
msgid ""
"Mastodon doesn't support display names longer than 30 characters, truncating "
"it and trying again..."
msgstr ""
"Mastodon no soporta nombres para mostrar de más de 30 caracteres, "
"truncándolo e intentándolo de nuevo..."

#: _utils.py:248
msgid "Mastodon doesn't support rich text. Disabling it..."
msgstr "Mastodon no soporta texto enriquecido. Desactivándolo..."

#: _utils.py:254
msgid "How far back should we retrieve tweets from the Twitter account?"
msgstr "¿Desde cuándo deberíamos recopilar los tweets de la cuenta de Twitter?"

#: _utils.py:257
msgid ""
"\n"
"Enter a date (YYYY-MM-DD):\n"
"[Leave it empty to retrieve *ALL* tweets or enter 'continue'\n"
"if you want the bot to execute as normal (checking date of \n"
"last post in the Fediverse account)] "
msgstr ""
"\n"
"Introduce una fecha (AAAA-MM-DD):\n"
"[Déjalo en blanco si quieres que se recopilen *TODOS*\n"
"los tweets ó introduce 'continue' si quieres que el bot se\n"
"comporte como de costumbre (comprobando la fecha del\n"
"último post de la cuenta del Fediverso)] "

#: cli.py:105
msgid ""
"Visibility not supported! Values allowed are: public, unlisted, private and "
"direct"
msgstr ""
"Visibilidad introducida no soportada. Valores válidos: public, unlisted, "
"private y direct"

#: cli.py:123
msgid "No Pleroma URL defined in config! [pleroma_base_url]"
msgstr ""
"¡No se ha definido la URL de Pleroma en la configuración! [pleroma_base_url]"

#: cli.py:198
msgid ""
"Some or all OAuth 1.0a tokens missing, falling back to application-only "
"authentication"
msgstr ""
"No todos los tokens OAuth 1.0a han sido encontrados. Usando autenticación de "
"aplicación en su lugar"

#: cli.py:231
msgid "Bot for mirroring one or multiple Twitter accounts in Pleroma/Mastodon."
msgstr "Bot para replicar una o varias cuentas de Twitter en Pleroma/Mastodon."

#: cli.py:244
msgid ""
"path of config file (config.yml) to use and parse. If not specified, it will "
"try to find it in the current working directory."
msgstr ""
"ruta del fichero de configuración (config.yml) a usar e interpretar. Si no "
"se especifica, se intentará usar el del directorio de trabajo actual."

#: cli.py:258
msgid ""
"skips Fediverse profile update (no background image, profile image, bio "
"text, etc.)"
msgstr ""
"omite la actualización de perfil en la cuenta del Fediverso (imagen de "
"fondo, imagen de perfil, biografía, etc.)"

#: cli.py:272
msgid ""
"forces the tweet retrieval to start from a specific date. The "
"twitter_username value (FORCEDATE) can be supplied to only force it for that "
"particular user in the config"
msgstr ""
"fuerza una fecha específica de comienzo para la recopilación de tweets. El "
"valor de twitter_username (FORCEDATE) puede ser introducido para forzar la "
"fecha de inicio de recopilación sólo para ese usuario específico del archivo "
"de configuración"

#: cli.py:285
msgid "skips first run checks"
msgstr "omite las validaciones de la primera ejecución"

#: cli.py:308
msgid "Debug logging enabled"
msgstr "Modo de depuración activado"

#: cli.py:326
msgid ""
"Multiple twitter users for one Fediverse account, skipping profile and "
"pinned tweet."
msgstr ""
"Varios usuarios de Twitter definidos para una cuenta de Fediverso, omitiendo "
"la actualización de perfil y tweet fijado."

#: cli.py:341
msgid "Processing user:\t{}"
msgstr "Procesando usuario:\t{}"

#: cli.py:347
msgid "It seems like pleroma-bot is running for the first time for this user"
msgstr ""
"Parece que pleroma-bot está ejecutándose por primera vez para este usuario"

#: cli.py:368
msgid ""
"Unable to retrieve tweets. Is the account protected? If so, you need to "
"provide the following OAuth 1.0a fields in the user config:\n"
" - consumer_key \n"
" - consumer_secret \n"
" - access_token_key \n"
" - access_token_secret"
msgstr ""
"No se han podido recopilar los tweets. ¿Está la cuenta protegida? Si lo "
"está, necesita proporcionar los siguientes valores de OAuth 1.0a en la "
"configuración:\n"
" - consumer_key \n"
" - consumer_secret \n"
" - access_token_key \n"
" - access_token_secret"

#: cli.py:378
msgid "tweet count: \t {}"
msgstr "número de tweets: \t {}"

#: cli.py:402
msgid "Multiple twitter users, not updating profile"
msgstr "Varios usuarios de Twitter, omitiendo actualizado de perfil"

#: cli.py:409
msgid "Exception occurred"
msgstr "Se produjo una excepción"
